chain/sub/incoming.go

```diff
@@ -473,19 +478,19 @@ func (bv *BlockValidator) checkPowerAndGetWorkerKey(ctx context.Context, bh *typ
 	baseTs := bv.chain.GetHeaviestTipSet()
 	lbts, err := stmgr.GetLookbackTipSetForRound(ctx, bv.stmgr, baseTs, bh.Height)
 	if err != nil {
-		log.Warnf("failed to load lookback tipset for incoming block")
-		return address.Undef, err
+		log.Warnf("failed to load lookback tipset for incoming block: %s", err)
+		return address.Undef, ErrSoftFailure
```

Comment:
Don't swallow errors into the log.
Use `xerrors.Errorf("something something: %w")` and `errors.Is(err, ErrSoftFailure)` above.

Reply:
sure, but we don't do anything with the error!